### ISSUE TITLE
NTR: cnida

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#13341]
I would like to move this term from uberon to go

```
[Term]
id: UBERON:0008277 ! cnida
name: cnida
def: "A giant secretory organelle that comprises a bulb-shape capsule containing a coiled hollow tubule structure attached to it. A cnida defines the phylum Cnidaria" [http://en.wikipedia.org/wiki/Cnida#Structure_and_function, http://orcid.org/0000-0002-6601-2165]
property_value: editor_note "Should be moved to GO. WP classifies as organelle but TOLweb says: A cnida therefore is technically not an organelle, but, rather, the most complex secretory product known" xsd:string
synonym: "cnidae" EXACT PLURAL []
is_a: UBERON:0000470 ! cell part
relationship: only_in_taxon NCBITaxon:6073 ! Cnidaria
```

There is no high priority for this term. I just want to have it tracked, now I can proceed with:

https://github.com/obophenotype/uberon/issues/1314

**Comment by pgaudet** [comment]
Hi Chris, 

Where should I put the ORCID? really in the def ? 
http://orcid.org/0000-0002-6601-2165

Thanks, Pascale

**Comment by pgaudet** [comment]
Hello again, 

Is it OK if this is a child of GO 'cell part' rather than UBERON 'cell part' ? 

Thanks, 
Pascale

**Comment by cmungall** [comment]
You can just change it to my dbxred for now. We should do a mass switch
sometime

**Comment by pgaudet** [comment]
The Taxon NCBITaxon:6073 ! Cnidaria is not available in Protégé. @ukemi wrote: 
"We used to be able to request imports in another file, but
since we have moved to Github, I don't think that file is valid any more.
I notice that many of the never_in taxon restriction objects are missing
from GO. I think there are issues with this import as well. "

What is the procedure to integrate new terms from external CVs?
